Makefile

```diff
@@ -34,6 +34,28 @@ dev-setup:
 dev-setup-extensions:
 	@./hack/dev-setup-extensions
 
+.PHONY: dev-setup-local-garden
+dev-setup-local-garden:
```

Comment:
It would be nice to also have a target to stop this nodeless cluster and remove the Docker containers and network.
WDYT of renaming this target to `local-cluster-up` and adding a second target `local-cluster-down` for removing the containers and the network?

Reply:
thanks for the suggestion, renamed to local-garden-up/down.